src/engine/shared/http.cpp

```diff
@@ -385,7 +385,7 @@ void CHttpRequest::OnCompletionInternal(void *pHandle, unsigned int Result)
 	{
 		if(g_Config.m_DbgCurl || m_LogProgress >= HTTPLOG::FAILURE)
 		{
-			log_error("http", "%s failed. libcurl error (%u): %s", m_aUrl, Code, m_aErr);
+			log_error("http", "%s failed. libcurl error (%u): %s", m_aUrl, Code, m_aErr[0] == '\0' ? curl_easy_strerror(Code) : m_aErr);
```

Comment:
```suggestion
			log_error("http", "%s failed. libcurl error (%u): %s", m_aUrl, Code, m_aErr[0] != '\0' ? m_aErr : curl_easy_strerror(Code));
```

Nit, not required for merging: Reads nicer for me, because then it's "if `m_aErr` contains something, use it, else …".